src/serialize.js

```diff
@@ -94,7 +94,7 @@ jQuery.fn.extend({
 			var type = this.type;
 
 			// Use .is( ":disabled" ) so that fieldset[disabled] works
-			return this.name && !jQuery( this ).is( ":disabled" ) &&
+			return this.name && !jQuery.find.matchesSelector( this, ":disabled" ) &&
```

Comment:
raw     gz Sizes
    255072  75469 dist/jquery.js
     84466  29360 dist/jquery.min.js
    
       raw     gz Compared to master @ 8e4aac8cb03ffb88373ea99629165d82ff5eccdd
       +16     +8 dist/jquery.js
       +17     +7 dist/jquery.min.js